app/src/main/java/org/oppia/android/app/topicdownloaded/TopicDownloadedActivity.kt

```diff
@@ -0,0 +1,76 @@
+package org.oppia.android.app.topicdownloaded
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import org.oppia.android.app.activity.InjectableAppCompatActivity
+import org.oppia.android.app.home.RouteToTopicListener
+import org.oppia.android.app.topic.TopicActivity
+import javax.inject.Inject
+
+private const val TOPIC_DOWNLOADED_ACTIVITY_TOPIC_ID_ARGUMENT_KEY =
+  "TopicDownloadedActivity.topic_id"
+private const val TOPIC_DOWNLOADED_ACTIVITY_INTERNAL_PROFILE_ID_ARGUMENT_KEY =
+  "TopicDownloadedActivity.internal_profile_id"
+
+/** The activity for displaying [TopicDownloadedFragment]. */
```

Comment:
The fragment is an implementation detail. Similar to my earlier activity KDoc comment, please rephrase this to explain what it's meant to show.